app/views/categories/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= render "layouts/navbar", title:"CATEGORIES" %>
```

Comment:
Categories page well implemented 💯 

✔️ For each category, the user can see their name, icon and the total amount of all the transactions that belongs to that category.
✔️ When the user clicks (or taps) on a category item, the application navigates to the transactions page for that category
✔️ There is a button "add a new category" at the bottom that brings the user to the page to create a new category

![image](https://github.com/kensteph/RoR-Capstone-Project/assets/38088818/b894a551-fce6-4d01-bd05-11ec13b02128)